packages/libresprite-git/libresprite-git.pacscript

```diff
@@ -1,7 +1,7 @@
 maintainer="echometerain <echometer@disroot.org>"
 
 name=libresprite-git
-version=1.2.1-Beta
+version=1.0-beta.1
```

Comment:
Use `pkgver` function.

Reply:
good idea

Reply:
Fixed in https://github.com/pacstall/pacstall-programs/pull/330/commits/d32a36add269b450478910f11524f04f32eefe50